trio/_subprocess/unix_pipes.py

```diff
@@ -0,0 +1,111 @@
+import os
+from typing import Tuple
+
+from .. import _core, BrokenStreamError
+from .._abc import SendStream, ReceiveStream
+
+__all__ = ["PipeSendStream", "PipeReceiveStream", "make_pipe"]
+
+
+class _PipeMixin:
+    def __init__(self, pipefd: int):
+        if not isinstance(pipefd, int):
+            raise TypeError(
+                "{0.__class__.__name__} needs a pipe fd".format(self)
+            )
+
+        self._pipe = pipefd
+        self._closed = False
+
+        import fcntl
```

Comment:
Can we move this import to the top of the file, like other imports?

Reply:
The import was originally there because windows tests would import test_unix_pipes and crash when fcntl was imported. I rearranged some stuff in the test so now this won't happen.